tests/metrics/test_usermetrics.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import tempfile
+import pandas as pd
+from typing import List
+
+from indico_toolkit.metrics import UserMetrics
+
+
+@pytest.fixture(scope="module")
+def ex_user_metrics_object(indico_client):
+    user_metrics = UserMetrics(indico_client)
+    return user_metrics
+
```

Comment:
I think we should add some tests for get_user_metrics_df as well

Reply:
We definitely need a test post refactor. You might want to mock some sample I/O. 

We need some kind of example of what the output data looks like too.

Reply:
Resolved here - https://github.com/IndicoDataSolutions/Indico-Solutions-Toolkit/blob/e49e5e55a1c12d9a696330b5f1355051839ed5b2/tests/metrics/test_usermetrics.py#L26